Firestore/Source/Remote/FSTDatastore.mm

```diff
@@ -319,7 +319,8 @@ - (void)invokeRPCWithFactory:(GRPCProtoCall * (^)(void))rpcFactory
             [FSTDatastore
                 prepareHeadersForRPC:rpc
                           databaseID:&self.databaseInfo->database_id()
-                               token:(result.is_valid() ? result.token() : absl::string_view())];
+                               token:(result.user().is_authenticated() ? result.token()
+                                                                       : absl::string_view())];
```

Comment:
FYI- I think this needs to be clang-formatted. I noticed it in my original commit, but if you clang-format, it reindents this whole block causing a bigger diff, so I didn't bother.  But once you merge this into mater, perhaps we should immediately follow up with a clang-format fixup PR or something.

Reply:
Oh! Actually I see you have a "fix lint" commit in here, so maybe you already took care of the issue.  If so, ignore me.